synapse/util/ratelimitutils.py

```diff
@@ -51,6 +52,34 @@ def new_limiter() -> "_PerHostRatelimiter":
             str, "_PerHostRatelimiter"
         ] = collections.defaultdict(new_limiter)
 
+        # We track the number of affected hosts per time-period so we can
+        # differentiate one really noisy homeserver from a general
+        # ratelimit tuning problem across the federation.
+        LaterGauge(
```

Comment:
How often does a `Collector` get its `collect` callback called? It's unclear to me how often these gauges are reported.

Reply:
Every time anything calls `/metrics`, which prometheus does ~15s.

Reply:
I think it's when it gets scraped, so it depends on what scrape interval you've set in Prometheus etc.

Reply:
Thanks for the info @erikjohnston and @reivilibre!